app/enterprise/0.36-x/index.md

```diff
@@ -1,5 +1,7 @@
 ---
 title: Kong Enterprise Documentation
+redirect_from:
+  - /Users/cat/docs.konghq.com/dist/enterprise/
```

Comment:
Do we know where this is originally linked?

Reply:
not sure but I can check with our SEO agency

Reply:
Cool. Not a blocker, so I'm going to merge.